src/main/golo/standard-augmentations.golo

```diff
@@ -284,6 +284,12 @@ augment java.util.Collection {
   ----
   function newWithSameType = |this| -> _newWithSameType(this)
 
+  ----
+  Destructuration helper.
+  
+  * return a tuple of the values
+  ----
+  function destruct = |this| -> Tuple.fromArray(this: toArray())
```

Comment:
We are here augmenting `Collection`. Or did you mean `Tuple copy(Collection<Object>)` ?